### DISCUSSION TITLE
Is there any way to disable every feature than the economy stuff?

### DISCUSSION THREAD

**Discussion by RafaeloxMC** [EssentialsX/Essentials#4908] [Q&A] [answered]
The title says everything. I want Essentials Economy with Vault only. No colored names, /gm or something

**Comment by osfanbuff63** [comment]
The best way to do this is just don't give permission for anything but the economy commands, and disable/enable any relevant config options. I don't think there's a way to disable features of the plugin entirely however.

**Comment by Chew** [comment, marked as answer]
If you're just wanting economy, you're much better off just finding a plugin that solely does economy...
